pom.xml

```diff
@@ -543,19 +543,19 @@
             <dependency>
                 <groupId>com.graphql-java</groupId>
                 <artifactId>graphql-java-tools</artifactId>
-                <version>4.3.0</version>
+                <version>5.2.4</version>
             </dependency>
 
             <dependency>
                 <groupId>com.graphql-java</groupId>
                 <artifactId>graphql-spring-boot-starter</artifactId>
-                <version>4.0.0</version>
+                <version>5.0.2</version>
             </dependency>
 
             <dependency>
                 <groupId>com.graphql-java</groupId>
                 <artifactId>graphiql-spring-boot-starter</artifactId>
-                <version>4.0.0</version>
+                <version>5.0.2</version>
```

Comment:
this dependency seems aligned with the one above.
therefore, the version number 5.0.2 should be put in a property, eg called `graphql.spring.version`